apps/user-office-backend/src/datasources/postgres/records.ts

```diff
@@ -791,11 +795,16 @@ export const createFileMetadata = (record: FileRecord) => {
   );
 };
 
-export const createQuestionTemplateRelationObject = (
+export const createQuestionTemplateRelationObject = async (
   record: QuestionRecord &
     QuestionTemplateRelRecord & { dependency_natural_key: string },
   dependencies: FieldDependency[]
 ) => {
+  let recordWithApiCall;
+  if (record.config.hasOwnProperty('url')) {
```

Comment:
I don't know if we have covered a situation where  'url' is not available .

Reply:
Yes, I included potential errors that I could think of. 
First check if it is a valid url address with regex on frontend
Then we check if api call fails or jsonpath is wrong syntax on backend. And return empty string on either of errors